PalaceTests/TPPBookCreationTestsObjc.m

```diff
@@ -1,38 +0,0 @@
-//
```

Comment:
Attempting to get this test to compile with its dependency on TPPBook.swift was problematic and it provides little value, so I removed it.